src/bilder/images/edxapp/templates/ansible_vars.j2.yml

```diff
@@ -0,0 +1,154 @@
+  STORAGE_KWARGS:
+    bucket: {{ bucket_prefix }}-edx-video-upload-{{ purpose }}-{{ environment }}
+  DIRECTORY_PREFIX: 'video-images/'
+EDXAPP_VIDEO_TRANSCRIPTS_SETTINGS:
+  VIDEO_TRANSCRIPTS_MAX_BYTES : 3145728
+  STORAGE_CLASS: 'storages.backends.s3boto.S3BotoStorage'
+  STORAGE_KWARGS:
+    bucket: {{ bucket_prefix }}-storage-{{ purpose }}-{{ environment }}
+  DIRECTORY_PREFIX: 'video-transcripts/'
+
+
+# Static Values
+COMMON_ENABLE_AWS_ROLE: false
+COMMON_ENABLE_BASIC_AUTH: false
+COMMON_ENABLE_MINOS: false
+COMMON_ENABLE_NEWRELIC: false
+COMMON_ENABLE_NEWRELIC_INFRASTRUCTURE: false
+EDXAPP_ANALYTICS_DASHBOARD_URL: !!null
+EDXAPP_BUGS_EMAIL: {{ smtp_config.data.support_email }}
+EDXAPP_CACHE_BACKEND: "django_redis.cache.RedisCache"
+EDXAPP_CELERY_BROKER_HOSTNAME: edx-redis.service.consul
+EDXAPP_CELERY_BROKER_TRANSPORT: 'redis'
+EDXAPP_CELERY_USE_SSL: true
+EDXAPP_CMS_MAX_REQ: 1000
+EDXAPP_CMS_NGINX_PORT: 80
+EDXAPP_CMS_SSL_NGINX_PORT: 443
+EDXAPP_COMMENTS_SERVICE_URL: "http://edx-forum.service.consul:4567"
+EDXAPP_CONTACT_EMAIL: {{ smtp_config.data.support_email }}
+EDXAPP_CUSTOM_COURSES_EDX: true
+EDXAPP_DEFAULT_FEEDBACK_EMAIL: "{{ smtp_config.data.support_email }}"
+EDXAPP_DEFAULT_FILE_STORAGE: 'storages.backends.s3boto.S3BotoStorage'
+EDXAPP_DEFAULT_FROM_EMAIL: "{{ smtp_config.data.support_email }}"
+EDXAPP_EMAIL_HOST: {{ smtp_config.data.relay_host }}
+EDXAPP_EMAIL_PORT: {{ smtp_config.data.relay_port }}
+EDXAPP_EMAIL_USE_TLS: true
+EDXAPP_ENABLE_COMBINED_LOGIN_REGISTRATION: true
+EDXAPP_ENABLE_CORS_HEADERS: true
+EDXAPP_ENABLE_CROSS_DOMAIN_CSRF_COOKIE: true
+EDXAPP_ENABLE_CSMH_EXTENDED: true
+EDXAPP_ENABLE_DISCUSSION_HOME_PANEL: true
+EDXAPP_ENABLE_GRADE_DOWNLOADS: true
+EDXAPP_ENABLE_LTI_PROVIDER: true
+EDXAPP_ENABLE_MOBILE_REST_API: true
+EDXAPP_ENABLE_OAUTH2_PROVIDER: true
+EDXAPP_ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES: false
+EDXAPP_ENABLE_SPECIAL_EXAMS: true
+EDXAPP_ENABLE_THIRD_PARTY_AUTH: true
+EDXAPP_GIT_REPO_DIR: "{{ edxapp_git_repo_dir }}"
+EDXAPP_GOOGLE_ANALYTICS_ACCOUNT: {{ edxapp_google_analytics_account }}
+EDXAPP_GRADE_STORAGE_TYPE: S3
+EDXAPP_INSTALL_PRIVATE_REQUIREMENTS: true
+EDXAPP_JWT_AUDIENCE: '{{ business_unit }}-{{ environment }}-key'
+EDXAPP_JWT_SIGNING_ALGORITHM: 'RS512'
+EDXAPP_LMS_MAX_REQ: 1000
+EDXAPP_MANAGE_MICROFRONTENDS: true
+EDXAPP_MEMCACHE: edx-redis.service.consul:6379
+EDXAPP_MONGO_DB_NAME: edxapp
+EDXAPP_MONGO_HOSTS: mongodb-master.service.consul
+EDXAPP_MONGO_PORTS: 27017
+EDXAPP_MONGO_REPLICA_SET: rs0
+EDXAPP_MONGO_USE_SSL: "{{ MONGODB_USE_SSL }}"
+EDXAPP_MYSQL_CSMH_DB_NAME: edxapp_csmh
+EDXAPP_MYSQL_CSMH_HOST: edxapp-mysql.service.consul
+EDXAPP_MYSQL_CSMH_PORT: {{ MYSQL_PORT }}
+EDXAPP_MYSQL_HOST: edxapp-mysql.service.consul
+EDXAPP_ORA2_FILE_PREFIX: "{{ salt.grains.get('environment') }}-dev/ora2"
+EDXAPP_PLATFORM_DESCRIPTION: MITx Online Course Portal
+EDXAPP_PLATFORM_NAME: MITx Online
+EDXAPP_SEARCH_HOST: elasticsearch.service.consul
+EDXAPP_SEARCH_USE_SSL: false
```

Comment:
why not?

Reply:
Because we don't currently have Elasticsearch for edX set up with TLS certs via PKI. It's on the roadmap though.